apps/public-docsite-v9/src/DocsComponents/ThemePicker.stories.tsx

```diff
@@ -32,15 +32,18 @@ const useStyles = makeStyles({
  */
 export const ThemePicker: React.FC<{ selectedThemeId?: string }> = ({ selectedThemeId }) => {
   const styles = useStyles();
+  const [currentThemeId, setCurrentThemeId] = React.useState(selectedThemeId ?? null);
```

Comment:
I dig a bit deeper and it looks like that `const context = React.useContext(DocsContext);` within `FluentDocsPage.stories` doesn't work at all - it has no context property. To properly fix this we should fix the issue  over there as this is moreless stateless component that is being set by parent.

Reply:
quickly looking at others custom controls we have and it seems we might not have another option. `TOC` is doing the same thing. 

Let's remove `{ selectedThemeId?: string }` prop API from `ThemePicker` as its not being set anyways and handle it internally.

Reply:
Actually the theme property is set in the global context after the emit, but because we can't really listen to those changes in an elegant way we still neet that `selectedThemeId` prop. This makes the dropdown keep the same value when switching from one component to another (ie change the theme on `<Button>`, then click on `<CompoundButton>`).

Reply:
it is set, but DocsPage wont get those values from globals thus this code doesn't do anything 

<img width="772" alt="image" src="https://user-images.githubusercontent.com/1223799/200361885-a7dfb549-ca3c-41a9-9ee6-655ae5d07be6.png">

This is the change  I had in mind + applying your changes in this PR 
![MicrosoftTeams-image](https://user-images.githubusercontent.com/1223799/200362048-b18a4d69-b40e-48e0-9dd0-a1187d6479a9.png)

Reply:
So what I mean is that the `FluentDocsPage` example where `ThemePicker` passes that `selectedTheme.id` value to the prop **is** actually working after the `addons.getChannel().emit('updateGlobals', { globals: { [THEME_ID]: themeId } });` call. This is the mechanism we use to keep the theme name consistent in the dropdown when changing pages.